GPU/GLES/GLES_GPU.cpp

```diff
@@ -565,6 +565,8 @@ void GLES_GPU::CheckGPUFeatures() {
 	if (gl_extensions.GLES3 || !gl_extensions.IsGLES)
 		features |= GPU_SUPPORTS_TEXTURE_LOD_CONTROL;
 
+	features |= GPU_SUPPORTS_ANISOTROPY;
```

Comment:
Hm, I think many mobile GPUs do not support aniso... it's not even standardized in ES 2.0, IIRC. We should do an extension check here.